src/main/resources/com/rsmart/kuali/kfs/fp/spring-fp.xml

```diff
@@ -100,7 +100,7 @@
 		<property name="vendorService" ref="vendorService"/>
 		<property name="batchFeedHelperService" ref="batchFeedHelperService"/>
 		<property name="attachmentsPath">
-	       <value>${staging.directory}/fp/disbursementVoucher</value>
+	       <value>${staging.directory}/fp/disbursementVoucher/attachment</value>
```

Comment:
Should this be attachments (plural) to be consistent with our main attachments directory?